setup.py

```diff
@@ -51,7 +51,12 @@
               'snapcraft.plugins',
               'snapcraft.storeapi'],
     package_data={'snapcraft.internal.repo': ['manifest.txt']},
-    scripts=['bin/snapcraft', 'bin/snapcraft-parser'],
+    entry_points={
+        'console_scripts': [
+            'snapcraft = snapcraft.cli.__main__:run',
+            'snapcraft-parser = snapcraft.internal.parser:main',
+        ],
+    },
```

Comment:
`scripts` are the old distutils way of doing things, `entry_points` are setuptools. They're the [recommended way of doing this](https://packaging.python.org/tutorials/distributing-packages/#scripts), particularly when you're talking [cross-platform stuff](https://setuptools.readthedocs.io/en/latest/setuptools.html#automatic-script-creation).